packages/web/src/components/Header/README.md

```diff
@@ -0,0 +1,272 @@
+
+## Accessibility Guidelines
+
+- Ensure accessibility by using a `<nav>` element or, if using a more generic
+  element such as a `<div>`, add a `role="navigation"` to every navigation to
+  explicitly identify it as a landmark region for users of assistive
+  technologies.
+- Indicate the current item by using `aria-current="page"` for the current
+  page or `aria-current="true"` for the current item in a set.
+- Don't forget to provide textual equivalents for icon buttons. Use either the
+  `aria-label` attribute, or the provided `accessibility-hidden` helper class.
+- For Header to work correctly with assistive technologies, the `aria-expanded`
+  attributes need to be updated by JavaScript when Header is toggled.
+
+👉 The animation effect of this component is dependent on the
+`prefers-reduced-motion` media query.
+
+## Minimal Header
+
+Without any modifier, Header is ready to contain necessary blocks in a classic
+left-to-right layout. Please note its fully transparent unless you add a color
+variant modifier class.
+
+```html
+<header class="Header">
+  <a href="#">
+    <img src="…" alt="Logo" />
+  </a>
+</header>
+```
+
+## Color Variants
+
+Currently, Header comes in two color variants: **transparent** (for dark
+backgrounds) and **inverted** (for light backgrounds). Add `Header--inverted`
+modifier class to make Header background inverted.
+
+```html
+<header class="Header Header--inverted">
+  <a href="#">
+    <img src="…" alt="Logo" />
+  </a>
+</header>
+```
+
+## Simple Header
+
+The `Header--simple` modifier makes the header bar slightly shorter and aligns
+its content to center. Use this design variant when all you need on the page is
+just branding.
+
+```html
+<header class="Header Header--simple">
+  <a href="#">
+    <img src="…" alt="Logo" />
+  </a>
+</header>
+```
+
+## Supported Content
+
+In order to create a responsive header with top-level navigation, there are the
+following building blocks to use:
+
+1. toggle button,
+2. actions bar with primary and secondary action slots,
+3. navigation,
+4. desktop dialog.
+
+### Toggle Button
+
+Toggle button reveals actions on small screens (up to the desktop breakpoint).
+It makes use of inverted Button by default.
+
+```html
+<div class="Header__toggle">
+  <button
+    type="button"
+    class="Button Button--inverted"
+    data-toggle="header"
+    data-target="#my-header-actions"
+    aria-controls="my-header-actions"
+    aria-expanded="false"
+  >
+    <span class="Header__icon Header__icon--menu" aria-hidden="true"></span>
+    <span class="accessibility-hidden">Menu</span>
+  </button>
+</div>
+```
+
+### Actions Bar
+
+Actions bar is a wrapper that holds one slot for primary actions (aligned to
+left), one slot for secondary actions (aligned to right), and a closing button.
+On small screens, actions are presented in form of an off-canvas pane sliding
+from right. From the desktop breakpoint up, the actions bar is rendered inside
```

Comment:
```suggestion
from the right. From the desktop breakpoint up, the actions bar is rendered inside
```

Reply:
all these grammar suggestions are based on Grammarly. So consider yourself if you want to add them.